tests/helpers/load_state_tests.py

```diff
@@ -335,6 +348,11 @@ def load_test(test_case: Dict, load: BaseLoad) -> Dict:
         json_data["blocks"]
     )
 
+    try:
+        post_state = load.json_to_state(json_data["postState"])
+    except KeyError:
+        raise NoPostState
```

Comment:
Not a huge blocker or anything, but this'll catch `KeyError`s from inside `json_to_state` as well. If the JSON format changes, we'll get xfails which might not be noticed.

I'd do something like:

```suggestion
    try:
        raw_post_state = json_data["postState"]
    except KeyError:
        raise NoPostState
    post_state = load.json_to_state(raw_post_state)
```
```